src/test/java/org/opensearch/security/dlic/rest/api/DashboardsInfoActionTest.java

```diff
@@ -22,28 +22,10 @@
 import org.opensearch.common.settings.Settings;
 import org.junit.Assert;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import java.util.Arrays;
 
-@RunWith(Parameterized.class)
```

Comment:
Why this is removed?

Reply:
because `kibanainfo` only works with ` /_opendistro/_security` and not `/_plugins/_security`.

Reply:
Where is the test for `/dashboardsinfo`?

Reply:
It seems that we do not have any test for it, I can add a test.

Reply:
I added test for `/dashboardsinfo`

Reply:
Why it is not parametrized?

Reply:
fixed